index.js

```diff
@@ -1,54 +1,80 @@
+const githubUsersApiEndPoint = 'https://api.github.com/users'
+const githubEndPoint = 'https://github.com'
 const addCollaborator = (collaborator) => {
   const list = document.querySelector('.list')
-  
+
   const collaboratorEl = document.createElement('div')
   collaboratorEl.className = 'collaborator'
-  
+
   const photo = document.createElement('img')
   const name = document.createElement('b')
   const role = document.createElement('span')
+
   collaboratorEl.appendChild(photo)
   collaboratorEl.appendChild(name)
   collaboratorEl.appendChild(role)
-  
   photo.src = collaborator.photo_url
   name.innerHTML = collaborator.name
   role.innerHTML = collaborator.role
-
+  
   list.appendChild(collaboratorEl)
+  if(collaborator.github){
+    const githubLink = document.createElement('a')
+    const githubIcon = document.createElement('img')
+    githubIcon.src = './assets/imgs/github.svg'
+    githubLink.appendChild(githubIcon)
+    githubLink.href = `${githubEndPoint}/${collaborator.github}`
+    githubLink.classList.add('contribIconLink')
+    collaboratorEl.insertAdjacentElement("beforeend",githubLink)
+  }
 }
 
 const collaborators = [
   {
     name: 'Winner Crespo',
     role: 'Propietario',
-    photo_url: 'https://media-exp1.licdn.com/dms/image/C4E03AQGBPcza7XsREQ/profile-displayphoto-shrink_200_200/0?e=1603324800&v=beta&t=4xbvuWqc5YmeYmsEXTtfNg_f9S3hzIf31LH48l8sbxc'
+    photo_url: 'https://media-exp1.licdn.com/dms/image/C4E03AQGBPcza7XsREQ/profile-displayphoto-shrink_200_200/0?e=1603324800&v=beta&t=4xbvuWqc5YmeYmsEXTtfNg_f9S3hzIf31LH48l8sbxc',
+    github: 'wistcc'
```

Comment:
`github` sounds too generic, what if we change the property name to something like `githubUsername` or something similar?